### PR TITLE
Updated layout for Speksit page

### PR DESCRIPTION
**Pull request by pymapa**

### CODE REVIEW

File: src/client/components/Speksit/Speksit.css [hunk 1]
```diff
@@ -97,100 +81,64 @@
...
-	.content_bratva {
-		padding-top: 30px;
-		padding-bottom: 30px;
-	}
-	.content_hali {
-		padding-top: 30px;
-		padding-bottom: 30px;
-	}
+	.speksiContainer {
+    flex-direction: row;
```
Comment (tatuheinonen) [inline src/client/components/Speksit/Speksit.css:98]: check indentation

File: src/client/components/Speksit/Speksit.css [hunk 2]
```diff
@@ -1,93 +1,77 @@
...
-	background-color: #FFECAF;
-	margin: 0px;
-	padding: 0px;
-	padding-top: 20px;
-	text-align: left;
+.whiteText {
+  /* color: white; */
 }
+
 .font_alaammuohi_header {
```
Comment (tatuheinonen) [inline src/client/components/Speksit/Speksit.css:48]: Should the style names be changed to match the show's year instead of name?

Comment (pymapa) [reply src/client/components/Speksit/Speksit.css:48]: Could be, but I think that isn't about this pr and can be done later when refactoring.

File: src/client/components/Speksit/Speksit.css (hunk 1 above)
Comment (pymapa) [reply src/client/components/Speksit/Speksit.css:98]: did some fixing, editor shows it correctly but some conflicts tab/2spaces exist.

**Review by tatuheinonen — APPROVED** [review]

**Pull request merged**